lua/framework/console/console.h

```diff
@@ -24,36 +24,35 @@ extern "C"
 /**IDK why this is needed. Seems like counter doesnt work properly without it*/
 #define ConsoleJoinWordsB(x, y) ConsoleJoinWordsA(x, y)
 
-/**Macro for registering a lua_CFunction within the Console
- * singleton, with the function being in the global namespace. Example:
+/**
+ * Macro for registering a `lua_CFunction` within the global namespace.
+ * Example:
  * \code
- * ConsoleRegisterLuaFunction(SolverInitialize);
+ * RegisterLuaFunction(SolverInitialize);
  * \endcode
  *
- * \note Remember to include the header "console/chi_console.h".
- * The name supplied to this function cannot have scope resolution operators,
- * i.e., "::".*/
-#define RegisterLuaFunctionAsIs(func_name)                                                         \
+ * \note Remember to include the header "lua/framework/console/console.h".
+ * The name supplied to this function cannot have scope resolution operators, i.e., "::".
+ *
+ * \param func_name LuaCFunction. The function to use.
+ */
+#define RegisterLuaFunction(func_name)                                                             \
   static char ConsoleJoinWordsB(unique_var_name_luacfunc_##func_name##_, __COUNTER__) =            \
     opensnlua::Console::AddFunctionToRegistryGlobalNamespace(#func_name, func_name)
 
-/**Macro for registering a lua_CFunction within the Console
-* singleton.
-\param function LuaCFunction. The function to use.
-\param namespace_name NonQuotedString. May include scope resolution
-\param func_name NonQuotedString. The name of the function as it will appear in
-                 the lua console.
-*/
-#define RegisterLuaFunctionNamespace(function, namespace_name, func_name)                          \
+/**
+ * Macro for registering a `lua_CFunction` within a namespace.
+ *
+ * \param function LuaCFunction. The function to use.
+ * \param namespace_name NonQuotedString. May include scope resolution
+ * \param func_name NonQuotedString. The name of the function as it will appear in lua.
```

Comment:
I think all of the comments may need to be looked at. I won't be marking them in the rest of the review but, if I am correct, please amend them

Reply:
An example of using this macro:
````
RegisterLuaFunctionInNamespace(LogSetVerbosity, log, SetVerbosity);
```
where `LogSetVerbosity` is the C++ function, i.e. `lua_CFunction`, `log` is the Lua namespace, `SetVerbosity` is the function name. Usage in Lua is `log.SetVerbosity`.

Where do you see any problems in the comment?

Reply:
If you expected more information, then please tell me what info you are looking for here as a user of these macros. I can add that, but you need to tell me...